src/router.jsx

```diff
@@ -5,10 +5,12 @@ import SignIn from './modules/sign-in/SignIn';
 import SignUp from './modules/sign-up/SignUp';
 import Feed from './modules/feed/Feed';
 import EmailVerify from './modules/email-verify/EmailVerify';
+import UserPage from './modules/user-page/UserPage';
 
 const Routes = (
   <BrowserRouter basename="/">
     <Switch>
+      <Route exact path='/user-page' component={UserPage} />
```

Comment:
strange name for a route. 'profile' would be better

Reply:
<Route exact path='/profile' component={UserPage} />

Reply:
done